core/tests/org.openjdk.jmc.flightrecorder.test/src/test/java/org/openjdk/jmc/flightrecorder/test/internal/util/ParserToolkitTest.java

```diff
@@ -0,0 +1,26 @@
+package org.openjdk.jmc.flightrecorder.test.internal.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openjdk.jmc.common.IMCFrame;
+import org.openjdk.jmc.flightrecorder.internal.util.ParserToolkit;
+
+public class ParserToolkitTest {
+	@Test
+	public void testParseBuiltinFrameType() {
+		Assert.assertTrue(IMCFrame.Type.INTERPRETED == ParserToolkit.parseFrameType(ParserToolkit.INTERPRETED_TYPE_ID));
+		Assert.assertTrue(
+				IMCFrame.Type.JIT_COMPILED == ParserToolkit.parseFrameType(ParserToolkit.JIT_COMPILED_TYPE_ID));
```

Comment:
Maybe make one line out of L12 and L13 😄

Reply:
Fixed